ocs_ci/ocs/ocs_upgrade.py

```
@@ -762,7 +762,16 @@ def set_upgrade_images(self):
             if not ocs_catalog.is_exist():
                 log.info("OCS catalog source doesn't exist. Creating new one.")
                 create_catalog_source(self.ocs_registry_image, ignore_upgrade=True)
-                prune_old_df_repo_idms()
+                if not config.ENV_DATA.get("platform", "").lower() in (
+                    constants.HCI_PROVIDER_CLIENT_PLATFORMS
+                ):
```

Ruff (0.15.18)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)